app/test.jsx

```diff
@@ -356,6 +355,14 @@ class App extends Component {
         this.setState({ markers });
     }
 
+    onWidget = () => {
+        const { enabledNavigationWidget } = this.state;
+
+        this.setState({
```

Comment:
Don't you think it's better to change it to the following?
`this.setState(prevState => ({ enabledNavigationWidget: !prevState.enabledNavigationWidget }))`

Reply:
@nazanin-binary nice shot, thanks.